hs_access_control/models.py

```diff
@@ -231,12 +342,16 @@ def create_group(self, title):
         owner.
         """
         if __debug__: 
-            assert isinstance(title, basestring) 
+            assert isinstance(title, basestring)
+            assert isinstance(description, basestring)
+            if purpose:
+                assert isinstance(purpose, basestring)
```

Comment:
@pkdash OK. It appears __debug__ constant is true when Python was not started with an -o option from python doc. In Django environment, I am assuming this __debug__ constant will be true when debug is set to true in django settings.py, but not sure. @alvacouch Can you comment on whether these assert statements can be removed from code base but instead checked in the method to return None or raise an exception message when input parameters are not valid?